app/src/main/java/com/example/storiesapp/ui/MainActivity.kt

```diff
@@ -48,79 +48,120 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         enableEdgeToEdge()
         setContentView(R.layout.activity_main)
+
+        viewModel = ViewModelProvider(this)[MainViewModel::class.java]
+
+        viewModel.getOrganizedList()
```

Comment:
Is `getOrganizedList()` a descriptive name? What I expect when I see `get` is that you get smth. So maybe smth like `prepare` instead of `getOrganized`? 
Smth else, `list` is not fully readable, what kind of list?